### PR TITLE
838 - Included the URL Validations for the stack

### PR DESCRIPTION
**Pull request by riginoommen**
# Resolves 838

# Explain the fix
Fixed the Url Validations for the component
## Screenshots
N/A

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] My changes generate no new warnings
- [x] I have performed a self-review of my own code
- [x] I have documented my code, particularly in areas like todo, complex logic, quick fix, temporary patch, etc.
#### If it is a new component
- [x] Does your component follow One Platform style guidelines?
- [x] Does your component web accessibility standards? [Helper Doc](https://www.w3.org/TR/wai-aria-1.1/)
- [x] Does your component support desktop screen sizes (350px, 720px, 1150px ,1920px)
#### Browsers you have tested in
- [x] Google Chrome [Latest 2 versions]

### CODE REVIEW

File: packages/opc-feedback/src/opc-feedback.ts [hunk 1]
```diff
@@ -88,7 +89,8 @@ export class OpcFeedback extends LitElement {
           category: (this._error) ? 'BUG' : 'FEEDBACK',
           stackInfo: {
             "stack": navigator.appVersion,
-            "path": window.location.pathname,
+            "path": ((this._path.length !== 1) && (this._path.substr(this._path.length-1, 1) === '/') || (this._path.substr(this._path.length-1, 1) === '#')) ? this._path.slice(0, -1) : this._path
```
Comment (gisumit) [inline packages/opc-feedback/src/opc-feedback.ts:92]: can you document what this validation does?

**Review by gisumit — CHANGES_REQUESTED** [review]

**Comment by hybridx** [comment]
@riginoommen are there more changes coming on the same component if so we shouldn’t keep deploying all the changes one by one but rather under a single version. If this is a final change then you might need to update the version of the component

File: packages/opc-feedback/src/opc-feedback.ts (hunk 1 above)
Comment (riginoommen) [reply packages/opc-feedback/src/opc-feedback.ts:92]: This actuallly truncates the last chracter `/` and `#` from end of URL.

Comment (gisumit) [reply packages/opc-feedback/src/opc-feedback.ts:92]: Ohh okay, can you add documentation?

**Review by gisumit — CHANGES_REQUESTED** [review]

**Review by hybridx — APPROVED** [review]

File: packages/opc-feedback/src/opc-feedback.ts (hunk 1 above)
Comment (riginoommen) [reply packages/opc-feedback/src/opc-feedback.ts:93]: Included the doc

**Review by gisumit — APPROVED** [review]

**Pull request merged**